Prac_8/dynamic_labels.py

```diff
@@ -0,0 +1,34 @@
+"""
+"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+from kivy.properties import StringProperty
+
+LIST = ["Justin", "Cat", "Dog", "Test", "Seven"]
+
+
+class DynamicLabels(App):
+    """"""
+    status_text = StringProperty()
+
+    def __init__(self, **kwargs):
+        """Construct main app."""
+        super().__init__(**kwargs)
+        self.names = LIST
+
+    def build(self):
+        """Build the Kivy GUI."""
+        self.title = "Dynamic Labels"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        self.create_widgets()
+        return self.root
+
+    def create_widgets(self):
```

Comment:
Nice code! btw, for my personal suggestion if naming the method as create_labels() instead of create_widgets() will be more straightforward, what do you think? haha